### DISCUSSION TITLE
Optimized profiles for Klipper?

### DISCUSSION THREAD

**Discussion by amithkk** [1coderookie/Klipper4KobraGoNeo#21] [Q&A] [answered]
With klipper installed, I'm assuming we can push the Stock Neo/Go hardware further to achieve faster print speeds. Can anyone who has experimented tell me what the "safe" limits are, and if you have slicer profiles optimized for use with klipper, share them too?

**Comment by 1coderookie** [comment, marked as answer]
First of all: sorry for my late reply! Somehow I missed it in the first place, and then I lost my phone and wasn't able to log in due to the 2FA.. :(  

The most limiting factor is the flowrate of the stock hotend, so that's what you need to keep an eye on. Once you reach the limit, you can try to increase it by using e.g. CHT nozzles or mod to a Volcano nozzle or whatever, but I personally didn't do so. I prefer quality and reliabilty over speed, so I always print on the slower side anyway, especially because most of the stuff I print I print with PETG.   

That being said, I'm not aware of optimized slicer profiles and besides that, imho it won't really help you much, because you have to dial in your settings for your specific setup and filaments anyway.
